v1/proto/service/gribi.pb.go

```diff
@@ -635,7 +687,15 @@ type ModifyRequest struct {
 	sizeCache     protoimpl.SizeCache
 	unknownFields protoimpl.UnknownFields
 
-	// A group of requests to add/modify/remove a single AFT entry
+	// A group of requests to add/modify/remove a single AFT entry.
```

Comment:
there are many changes to the comments but they're not in the .proto file corresponding to this.
Where are these changes coming from? I assume this file hasn't been hand edited

Reply:
resolving this, they're coming from prior merges that were comment only.